sunpy/visualization/colormaps/cm.py

```diff
@@ -29,6 +29,19 @@
 sohoeit284 = ct.eit_color_table(284*u.angstrom)
 sohoeit304 = ct.eit_color_table(304*u.angstrom)
 
+# Solar Orbiter EUI
+# These are deliberately the same as AIA
```

Comment:
out of curiosity - are these deliberately chosen here for sunpy or deliberately chosen by the instrument teams?

Reply:
I got in touch with the EUI team, and these are what they recommended; see https://github.com/sunpy/sunpy/issues/4703#issuecomment-741888574 for more info.

Reply:
oh yeah thanks!